src/stories/avatarLabelGroup.stories.tsx

```diff
@@ -11,7 +11,7 @@ const account: AccountItem = {
   id: '1',
   type: 'users',
   name: 'first last',
-  username: '@username',
+  username: 'username',
```

Comment:
@marjan-ara 

Not related to this PR. Please remove it from this PR.

Reply:
@marjan-ara next time, please make a new branch and keep only relevant lines of code. This time, I'm approving this.